server/src/data-layer/models/firebase.ts

```diff
@@ -1,22 +1,74 @@
 import { Timestamp } from "firebase-admin/firestore"
 
 export interface UserAdditionalInfo {
-  date_of_birth: Timestamp // Assuming this is a timestamp
+  /**
+   * @isDateTime Please Enter in the form dd/mm/yyyy
+   */
+  date_of_birth: Timestamp
+  /**
+   * @isbool Please enter true or false
+   */
   does_freestyle: boolean
+  /**
+   * @isbool Please enter true or false
+   */
   does_racing: boolean
+  /**
+   * @isbool Please enter true or false
+   */
   does_ski: boolean
+  /**
+   * @isString Please enter your pronouns
+   */
+  gender: string
```

Comment:
since you added a new field here, you will need to update the user mock (`User.mock.ts`) so the tests will pass